app/views/users/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>User Details</h1>
+<%= render partial: "user_card" , locals: { user: @user } %>
+  <%= render partial: "bio_card" , locals: { user: @user } %>
+
+    <h2>Last three posts:</h2>
+
+    <div class="card-list">
+      <% @recent_posts.each do |post| %>
+        <%= render partial: "posts/post_card" , locals: { post: post } %>
+          <% end %>
+    </div>
+
+    <%= link_to "See all posts" , user_posts_path(@user) if @user.posts_counter> 0 %>
+      <%= link_to "Add a new post" , new_user_post_url(@user) %>
+
+        <%=form_with model: User.new, url: destroy_user_session_path, method: :delete, data: { turbo: false }  do %> <button class="custom-button">Sign out</button>
+          <% end %>
```

Comment:
- [ ] Clicking on a single user in the `index` page should route you to that user's `show` page which should use the `#recent_post` method to render the user's 3 most recent posts.
This behaviour doesn't work exactly as expected right now and as such, I can't check other features like the user posts index. Please fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/179354397-5c60ed60-56d3-4bf8-84ec-2eee90514787.png)